include/LMS1xx/MRS1000.h

```diff
@@ -0,0 +1,16 @@
+#ifndef MRS1000_H
+#define MRS1000_H
+#include <LMS1xx/LMS1xx.h>
+
+class MRS1000 : public LMS1xx
```

Comment:
this cries for an abstract base class that both - LMS1xx and MRS1000 - could derive from...